routes/users.js

```diff
@@ -15,8 +15,8 @@ mongoose.connect(process.env.DB, {
 
 /* POST api/user/register */
 router.post('/register', (req, res) => {
-  const { username } = req.body;
-  User.find({ username }, (err, user) => {
+  const { username, email } = req.body;
+  User.find({ $or: [{ username }, { email }] }, (err, user) => {
```

Comment:
I think you will need to check `username` and then check `email` then if either one were found return an object like:
```js
res.send({
  usernameExists: true,
  emailExists: false,
)}
```